discovery-provider/src/utils/prometheus_metric.py

```diff
@@ -249,9 +266,8 @@ def save(self, value, labels=None):
         this_metric = self.metric
         if labels:
             this_metric = this_metric.labels(**labels)
-
         if isinstance(this_metric, Histogram):
-            this_metric.observe(value)
+            this_metric.observe(value, labels)
```

Comment:
Let's huddle over this once you get to a good point please?

It'd be good to know what you were seeing especially since the rest of the currently generated metrics were able to have labels assigned without this change:

* https://discoveryprovider2.staging.audius.co/prometheus_metrics

Reply:
Thanks for calling this out!

I've created [INF-241](https://linear.app/audius/issue/INF-241) to track this issue.

Let's run with your change for now, thanks!